src/components/claims/claim-details/components/ClaimPayment.tsx

```diff
@@ -37,284 +30,217 @@ export interface PaymentFormData {
   message?: string
 }
 
-const PaymentForm = styled(Form)`
-  margin-top: 1rem;
-`
-
-const SubmitButton = styled(MuiButton)`
-  && {
-    margin-top: 1rem;
-    display: block;
-  }
-`
-
-const Checkbox: React.FC<FieldProps> = ({
-  field: { onChange, onBlur, name, value },
-}) => (
-  <MuiCheckbox
-    onChange={onChange}
-    onBlur={onBlur}
-    name={name}
-    checked={value || false}
-    color="primary"
-  />
-)
-
 const areSwishPayoutsEnabled = () => {
   return (window as any).HOPE_FEATURES?.swishPayoutsEnabled ?? false
 }
 
-const getPaymentValidationSchema = (isPotentiallySanctioned: boolean) =>
-  yup.object().shape({
-    ...(isPotentiallySanctioned && {
-      overridden: yup
-        .boolean()
-        .required()
-        .test(
-          'overridden',
-          'Override sanction list checkbox isn’t checked.',
-          (value) => value === true,
-        ),
-    }),
-    amount: yup.string().required(),
-    note: yup
-      .string()
-      .min(5) // for some reason payment service enforces at least 5 chars in the note
-      .required(),
-    exGratia: yup.boolean(),
-    type: yup
-      .string()
-      .oneOf(['Automatic', 'AutomaticSwish', 'IndemnityCost', 'Expense'])
-      .required(),
-    phoneNumber: yup.string().when('type', {
-      is: 'AutomaticSwish',
-      then: yup.string().required(),
-      otherwise: yup.string().notRequired(),
-    }),
-    message: yup.string().when('type', {
-      is: 'AutomaticSwish',
-      then: yup.string().required(),
-      otherwise: yup.string().notRequired(),
-    }),
-  })
-
 export const ClaimPayment: React.FC<{
   sanctionStatus?: SanctionStatus | null
   claimId: string
   refetch: () => Promise<any>
   identified: boolean
   market: string
   carrier: string
-}> = ({ sanctionStatus, claimId, refetch, identified, market, carrier }) => {
-  const [createPayment, createPaymentProps] = useCreateClaimPaymentMutation()
-  const [
-    createSwishPayment,
-    createSwishPaymentProps,
-  ] = useCreateSwishClaimPaymentMutation()
+}> = ({ sanctionStatus, carrier, claimId, identified, market, refetch }) => {
+  const [createPayment] = useCreateClaimPaymentMutation()
+  const [createSwishPayment] = useCreateSwishClaimPaymentMutation()
 
   const [isConfirming, setIsConfirming] = useState(false)
-
-  const [paymentStatus, setPaymentStatus] = useState<
-    'COMPLETED' | 'FAILED' | null
-  >(null)
+  const [isExGratia, setIsExGratia] = useState(false)
+  const [isOverridden, setIsOverridden] = useState(false)
+
+  const categoryOptions = [
+    ...Object.keys(ClaimPaymentType).map((el, idx) => ({
```

Comment:
I'd say it's better to have verbose variable names (not too long of course, but this should be fine and very readable IMO)
```suggestion
    ...Object.keys(ClaimPaymentType).map((paymentType, index) => ({
```